<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="sidebar_frame">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_dashboard">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Escritorio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_nuevo_ticket">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Nuevo ticket</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="content_frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>9</y>
        <width>311</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Página 1</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_nombre">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>113</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_enviar_datos">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>180</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Enviar datos</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Página 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_bienvenida">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_volver">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Volver</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
